omod/src/main/resources/config.xml

```diff
@@ -23,10 +23,10 @@
 	</description>
 
 	<activator>${project.parent.groupId}.${project.parent.artifactId}.FhirActivator</activator>
-	
+
 	<!-- <updateURL>https://modules.openmrs.org/modules/download/@MODULE_ID@/update.rdf</updateURL> -->
 	<!-- /Base Module Properties -->
-	
+
```

Comment:
@ShivanshRakesh  we came out with a few unwanted code formats, mvn clean package could always help you clean up stuff before pushing

Reply:
@HerbertYiga I actually did `mvn clean package`, don't know why this happened. I'll push again to remove these unwanted changes.